### DISCUSSION TITLE
Free sql  group by 不能 .page

### DISCUSSION THREAD

**Discussion by fengzijun** [dotnetcore/FreeSql#1879] [General]
如题 ，group by 后只能 .TOLIST 不能 .page ,请问这个怎么解决 ？

**Comment by 2881099** [comment]
group by 后分页性能很低，如果非要分页，可以GroupBy().WithTempQuery().Page().ToList()

具体参考嵌套查询文档。

**Reply by fengzijun** [reply]
好的谢谢
